sdk/include/psdk/shlobj.h

```diff
@@ -197,6 +197,14 @@ SHGetFolderPathAndSubDirW(
   _In_opt_ LPCWSTR,
   _Out_writes_(MAX_PATH) LPWSTR);
 
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
+HRESULT WINAPI SHGetKnownFolderPath(
+  _In_ REFKNOWNFOLDERID rfid,
+  _In_ DWORD dwFlags,
+  _In_ HANDLE hToken,
```

Comment:
```suggestion
  _In_opt_ HANDLE hToken,
```
the parameter is optional (in the sense that it can be NULL)